### PR TITLE
Displayed order and items count on find order page and virtual and variant count on the product inventory page(#2838bb1)

### PR DESCRIPTION
**Pull request by disha1202**
### Related Issues

Closes #

### Screenshots of Visual Changes before/after (If There Are Any)

**IMPORTANT NOTICE** - Remember to add changelog entry

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/hotwax/commerce-hub#contribution-guideline)

### CODE REVIEW

File: src/views/FindOrder.vue [hunk 1]
```diff
@@ -254,6 +254,13 @@ export default defineComponent ({
     }
   },
   methods: {
+    getTotalItems(orders: any){
```
Comment (ymaheshwari1) [inline src/views/FindOrder.vue]: We can remove this method as it's unused.

File: src/store/modules/order/OrderState.ts [hunk 2]
```diff
@@ -1,7 +1,8 @@
 export default interface OrderState {
   list: {
     orders: any[],
-    total: number
+    total: number,
+    items: number
```
Comment (ymaheshwari1) [inline src/store/modules/order/OrderState.ts]: I think we can name it something like, total -> ordersCount, items -> itemsCount

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/store/modules/order/OrderState.ts (hunk 2 above)
Comment (ymaheshwari1) [reply src/store/modules/order/OrderState.ts]: @adityasharma7 Sir WDYT?

Comment (dt2patel) [reply src/store/modules/order/OrderState.ts]: The noun does not have to be plural when naming variables. When naming a variable the noun should be universal, order is always "order", not "orders".

Comment (ymaheshwari1) [reply src/store/modules/order/OrderState.ts]: Ok Sir

File: src/views/FindOrder.vue (hunk 1 above)
Comment (disha1202) [reply src/views/FindOrder.vue]: Removed

File: src/store/modules/order/OrderState.ts (hunk 2 above)
Comment (disha1202) [reply src/store/modules/order/OrderState.ts]: Improved variable names.

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**